play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/PublishBundle.kt

```diff
@@ -1,28 +1,36 @@
 package com.github.triplet.gradle.play.tasks
 
+import com.github.triplet.gradle.common.utils.safeCreateNewFile
 import com.github.triplet.gradle.play.PlayPublisherExtension
 import com.github.triplet.gradle.play.tasks.internal.PublishArtifactTaskBase
 import com.github.triplet.gradle.play.tasks.internal.PublishableTrackExtensionOptions
 import com.github.triplet.gradle.play.tasks.internal.workers.PublishArtifactWorkerBase
+import com.github.triplet.gradle.play.tasks.internal.workers.copy
 import com.github.triplet.gradle.play.tasks.internal.workers.paramsForBase
+import org.gradle.api.file.ConfigurableFileCollection
+import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.file.RegularFileProperty
-import org.gradle.api.tasks.InputFile
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.tasks.InputFiles
 import org.gradle.api.tasks.Optional
 import org.gradle.api.tasks.OutputDirectory
 import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
+import org.gradle.api.tasks.SkipWhenEmpty
 import org.gradle.api.tasks.TaskAction
 import org.gradle.kotlin.dsl.submit
 import org.gradle.kotlin.dsl.support.serviceOf
 import org.gradle.workers.WorkerExecutor
+import java.io.File
 import javax.inject.Inject
 
 internal abstract class PublishBundle @Inject constructor(
         extension: PlayPublisherExtension
 ) : PublishArtifactTaskBase(extension), PublishableTrackExtensionOptions {
     @get:PathSensitive(PathSensitivity.RELATIVE)
-    @get:InputFile
-    internal abstract val bundle: RegularFileProperty
+    @get:SkipWhenEmpty
+    @get:InputFiles
+    internal abstract val bundles: ConfigurableFileCollection
```

Comment:
This task can now upload multiple bundle files, similar to the `PublishApk` task.